lighthouse-core/lib/asset-saver.js

```diff
@@ -139,9 +139,8 @@ async function prepareAssets(artifacts, audits) {
 }
 
 /**
- * Generates a JSON representation of traceData line-by-line to avoid OOM due to very large traces.
- * COMPAT: As of Node 9, JSON.parse/stringify can handle 256MB+ strings. Once we drop support for
- * Node 8, we can 'revert' PR #2593. See https://stackoverflow.com/a/47781288/89484
```

Comment:
even though we *can* do this now, I think we like the one trace event per line format, so it made sense to drop the old comment.

Reply:
yes, we love it